autoload/lsp/utils/workspace_edit.vim

```diff
@@ -14,11 +15,12 @@ function! lsp#utils#workspace_edit#apply_workspace_edit(workspace_edit) abort
         let l:cur_buffer = bufnr('%')
         let l:view = winsaveview()
         for [l:uri, l:text_edits] in items(a:workspace_edit['changes'])
-            call lsp#utils#text_edit#apply_text_edits(l:uri, l:text_edits)
+            call lsp#utils#text_edit#apply_text_edits(l:uri, l:text_edits, {'show_edits': 1})
         endfor
         if l:cur_buffer !=# bufnr('%')
             execute 'keepjumps keepalt b ' . l:cur_buffer
         endif
         call winrestview(l:view)
     endif
+    execute 'lopen'
```

Comment:
Just wondering if we should open the location window automatically, as it may be annoying if you don't expect it?